test/graphemetest.c

```diff
@@ -23,8 +23,9 @@ void checkline(const char *_buf, bool verbose) {
             bi += 1;
         }
         else { /* hex-encoded codepoint */
-            size_t len = encode((unsigned char*) (src + si), buf + bi) - 1;
-            while (src[si]) ++si; /* advance to NUL termination */
```

Comment:
Did this stop working because one of the grapheme tests included a NUL character?